package.json

```diff
@@ -50,6 +48,7 @@
     "mocha": "2.4.5",
     "mocha-eslint": "2.0.2",
     "mocha-jsdom": "1.1.0",
+    "mz": "2.4.0",
```

Comment:
Any advantages of `mz` over `fs-promise`? They both look popular, just familiar with fs-promise